### PR TITLE
Question explanation title fix

### PR DESCRIPTION
**Pull request by huangjosh**
Use "Answer Submitted" as the explanation title if text response question is not autoGradable

### CODE REVIEW

File: client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx [hunk 1]
```diff
@@ -146,7 +146,11 @@ class SubmissionEditStepForm extends Component {
 
       let title = '';
       if (explanation.correct) {
-        title = intl.formatMessage(translations.correct);
+        if (question.type === questionTypes.TextResponse && !question.autoGradable) {
```
Comment (allenwq) [inline client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx]: I think this should be adopted to all non-autogradable questions (file upload, etc), what do you think?

Comment (huangjosh) [reply client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx]: That sounds better. Let me improve this.

**Comment by fonglh** [comment]
Really minor nitpick, please capitalise the 1st word in the commit message to be consistent with everyone else's.

**Comment by huangjosh** [comment]
@fonglh @allenwq Fixed, and updated the PR.

File: app/views/course/assessment/question/scribing/_scribing.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1 @@
+json.autogradable question.auto_gradable?
```
Comment (jeremyyap) [inline app/views/course/assessment/question/scribing/_scribing.json.jbuilder]: Just noticed this, missing newline :scream:

Comment (huangjosh) [reply app/views/course/assessment/question/scribing/_scribing.json.jbuilder]: OK, will be careful next time. we may fix this in next PR. 😨

**Pull request merged**